packages/integrations/cloudflare/README.md

```diff
@@ -107,6 +107,12 @@ export function get({ params }) {
 }
 ```
 
+## Headers, Redirects and function invocation routes
+
+Cloudflare has support for adding custom [headers](https://developers.cloudflare.com/pages/platform/headers/), configuring static [redirects](https://developers.cloudflare.com/pages/platform/redirects/) and defining which routes should [invoke functions](https://developers.cloudflare.com/pages/platform/functions/function-invocation-routes/). This is handled via the special files called _headers, _redirects and _routes.json which can be placed in the `public` folder to take effect.
+
+The _routes.json file is generated during build to prevent a function invocation on assets paths. This will be overwritten when a custom _routes.json is provided. If that is the case the static files will still be served, but the request will go through a function invocation and will count against the request limits of your Cloudflare plan.
```

Comment:
Have I understood this correctly? I’d suggest making this clearer that the adapter has a default behaviour which gets overridden by any custom file. I’m also suggesting a subheading to help highlight this specific case:

```suggestion
### Custom `routes.json`

By default, `@astrojs/cloudflare` will generate a `_routes.json` file for you to prevent function invocations for static assets. Creating a custom `_routes.json` will override this default behaviour. In this case, static assets will still be served, but the request will go through a function invocation and will count against the request limits of your Cloudflare plan.
```

Reply:
@delucis I have rewritten this section, to include your other remark as well. It is more descriptive now on what is happening.